### PR TITLE
Introduce new changelog page

### PR DESCRIPTION
**Pull request by gbossert**
This PR introduces a new changelog button that opens the SleekPlan widget with the changelog displayed.
The button includes a counter of the number of unseen changes since the last visit of the user.

image

### CODE REVIEW

File: src/sekoiaio.scss [hunk 1]
```diff
@@ -322,3 +326,44 @@
...
+    width: 3px;
+    z-index: 0;
+}
+
+.changelog .column-left {
+  flex: 15%;
+}
+.changelog .column-right {
+  flex: 85%;
+}
```
Comment (otetard) [inline src/sekoiaio.scss:369]: Just SCSS refactoring

```suggestion
.changelog {
    .row {
        display: flex;
        padding: 24px 64px;
        border-left: 1px solid #d1d5da;
        position: relative;
        padding: 24px 20px 24px 58px;

        &::before {
            transform: translateY(-50%);
            background: #d1d5da;
            content: "";
            left: 0;
            height: 1px;
            position: absolute;
            top: 50%;
            transform: translateY(-50%);
            width: 32px;
            z-index: 1;
        }

        &:last-child::after{
            background: #fff;
            content: "";
            height: 50%;
            position: absolute;
            left: -1px;
            bottom: 0;
            width: 3px;
            z-index: 0;
        }
    }

    .column-left {
        flex: 15%;
    }

    .column-right {
        flex: 85%;
    }
}
```

File: src/sekoiaio.scss [hunk 2]
```diff
@@ -99,10 +99,14 @@
...
-        a, a:hover {
-            color: #5351eb;
+        a {
+            color: #12005e;
         }
 
+	a:hover {
+	    color: #5351eb;
+	}
+
```
Comment (otetard) [inline src/sekoiaio.scss:109]: SCSS refactoring

```suggestion
a {
    color: #12005e;

    &:hover {
	color: #5351eb;
    }
}

```

File: docs/changelog/new-intelligence-souce-hatching-triage.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+title: New intelligence source: Hatching Triage
+hide: navigation
+  
+**July 2, 2021**
+# New intelligence source: Hatching Triage
```
Comment (otetard) [inline docs/changelog/new-intelligence-souce-hatching-triage.md:5]:
```suggestion
# New Intelligence Source: Hatching Triage
```

File: docs/changelog/sekoia-io-new-visual-identity.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+title: SEKOIA.IO's new visual identity!
+hide: navigation
+  
+**July 5, 2021**
+# SEKOIA.IO's new visual identity!
```
Comment (otetard) [inline docs/changelog/sekoia-io-new-visual-identity.md:5]:
```suggestion
# SEKOIA.IO’s New Visual Identity!
```

File: docs/changelog/sekoia-io-new-visual-identity.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+title: SEKOIA.IO's new visual identity!
```
Comment (otetard) [inline docs/changelog/sekoia-io-new-visual-identity.md:1]:
```suggestion
title: SEKOIA.IO’s New Visual Identity!
```

File: docs/changelog/new-intelligence-souce-hatching-triage.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+title: New intelligence source: Hatching Triage
```
Comment (otetard) [inline docs/changelog/new-intelligence-souce-hatching-triage.md:1]:
```suggestion
title: New Intelligence Source: Hatching Triage
```

File: docs/changelog/user-center-invitations.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+title: New invitation feature in the User Center
+hide: navigation
+  
+**July 8, 2021**
+# New invitation feature in the User Center
```
Comment (otetard) [inline docs/changelog/user-center-invitations.md:5]:
```suggestion
# New Invitation Feature in the User Center
```

File: docs/changelog/user-center-invitations.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+title: New invitation feature in the User Center
```
Comment (otetard) [inline docs/changelog/user-center-invitations.md:1]:
```suggestion
title: New Invitation Feature in the User Center
```

File: docs/changelog/user-center-invitations.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+hide: navigation
+  
+**July 8, 2021**
+# New invitation feature in the User Center
+
+The change of the invitation process makes it easier to : 
+
+- Assigning multiple roles to users: Different roles can be assigned at the same time when sending the invitation.
```
Comment (otetard) [inline docs/changelog/user-center-invitations.md:9]:
```suggestion
- Assigning multiple roles to users: different roles can be assigned at the same time when sending the invitation.
```

File: docs/changelog/user-center-invitations.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+- Adding a user known to SEKOIA.IO: You can add existing users in other communities directly to a new one without going through the email invitation and authentication process.
```
Comment (otetard) [inline docs/changelog/user-center-invitations.md:10]:
```suggestion
- Adding a user known to SEKOIA.IO: you can add existing users in other communities directly to a new one without going through the email invitation and authentication process.
```

File: theme/overrides/changelog.html [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/tracking-chinese-apt-groups">
+	Tracking of Chinese APT groups
+      </a>
+    </div>
+  </article>
+    
+</container>
```
Comment (otetard) [inline theme/overrides/changelog.html:64]: Can’t we make this section generated? We should probably write a small MkDocs plugin to do so.

File: theme/overrides/changelog.html [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+<container class="changelog">
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 8, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/user-center-invitations">
```
Comment (otetard) [inline theme/overrides/changelog.html:22]:
```suggestion
      <a href="changelog/user-center-invitations">
```

File: theme/overrides/changelog.html [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 5, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/sekoia-io-new-visual-identity">
```
Comment (otetard) [inline theme/overrides/changelog.html:34]:
```suggestion
      <a href="changelog/sekoia-io-new-visual-identity">
```

File: theme/overrides/changelog.html [hunk 14]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 2, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/new-intelligence-souce-hatching-triage">
```
Comment (otetard) [inline theme/overrides/changelog.html:46]:
```suggestion
      <a href="changelog/new-intelligence-souce-hatching-triage">
```

File: theme/overrides/changelog.html [hunk 15]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">June 28, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="/changelog/tracking-chinese-apt-groups">
```
Comment (otetard) [inline theme/overrides/changelog.html:58]:
```suggestion
      <a href="changelog/tracking-chinese-apt-groups">
```

**Review by otetard — COMMENTED** [review]
Nice work! A few typo suggestions and CSS refactoring.

File: theme/overrides/changelog.html (hunk 11 above)
Comment (Darkheir) [reply theme/overrides/changelog.html:64]: Having it automatically based on the title and the date in the changelog files would be nice indeed.

File: theme/overrides/changelog.html [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+
+<container class="changelog">
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 8, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="changelog/user-center-invitations">
```
Comment (Darkheir) [inline theme/overrides/changelog.html:22]:
```suggestion
      <a href="user-center-invitations">
```

File: theme/overrides/changelog.html [hunk 17]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 5, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="changelog/sekoia-io-new-visual-identity">
```
Comment (Darkheir) [inline theme/overrides/changelog.html:34]:
```suggestion
      <a href="sekoia-io-new-visual-identity">
```

File: theme/overrides/changelog.html [hunk 18]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">July 2, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="changelog/new-intelligence-souce-hatching-triage">
```
Comment (Darkheir) [inline theme/overrides/changelog.html:46]:
```suggestion
      <a href="new-intelligence-souce-hatching-triage">
```

File: theme/overrides/changelog.html [hunk 19]
```diff
@@ -0,0 +1,66 @@
...
+    </div>
+  </article>
+  
+  <article class="row">
+    <div class="column-left">
+      <span class="alt-mono-font">June 28, 2021</span>
+    </div>
+
+    <div class="column-right">
+      <a href="changelog/tracking-chinese-apt-groups">
```
Comment (Darkheir) [inline theme/overrides/changelog.html:58]:
```suggestion
      <a href="tracking-chinese-apt-groups">
```

**Review by Darkheir — COMMENTED** [review]
Theme looks good, only the link to the details posts seem broken right now 

`http://....ovh.net/.../182/changelog/changelog/user-center-invitations` instead of `http://....ovh.net/.../182/changelog/user-center-invitations`

File: docs/javascript/sekoiaio.js [hunk 20]
```diff
@@ -5,4 +5,11 @@ window.addEventListener("DOMContentLoaded", (event) => {
         let p = document.createElement("span");
         div.prepend(p);
     });
+
+    var uls = document.getElementsByClassName("md-tabs__list");
+    if (uls.length != 0) {
+	uls[0].insertAdjacentHTML("beforeend", "<li class='md-tabs__item'><a data-sleek data-badge-changelog class='md-tabs__link' href='#'>Changelog</a></li>");
+    }
```
Comment (Gwe-n) [inline docs/javascript/sekoiaio.js:12]:
```suggestion
    const ul = document.querySelector('.md-tabs__list');
    ul?.insertAdjacentHTML("beforeend", "<li class='md-tabs__item'><a data-sleek data-badge-changelog class='md-tabs__link' href='#'>Changelog</a></li>");
```

**Comment by mortrevere** [comment]
The search bar at the top of the page vanishes on `hover`

**Comment by otetard** [comment]
> The search bar at the top of the page vanishes on `hover`

This issue is not specific to this PR. I didn't manage to fix it, maybe @jadeen can have a look at it ;).

**Review by Darkheir — APPROVED** [review]
:rocket:

**Pull request merged**